### PR TITLE
PMM-12985 Apply v3 API infra modifications and update docs

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-12985

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [X] API Docs updated

### CODE REVIEW

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -29,11 +29,12 @@ jobs:
...
           echo "ID=$ID" >> $GITHUB_ENV
 
-      - name: Provision rdme
-        run: npm install -g rdme
-
       - name: Sync API spec
-        run: rdme openapi ./api/swagger/swagger.json --id=${{ env.ID }} --key=${{ secrets.README_TOKEN }}
+        uses: readmeio/rdme@v8
+        with:
+          rdme: openapi ./api/swagger/swagger.json --id=${{ env.ID }} --key=${{ secrets.README_TOKEN }}
```
Comment (BupycHuk) [inline .github/workflows/doc.yml:35]: should we pass --version flag here?

**Review by BupycHuk — APPROVED** [review]

File: .github/workflows/doc.yml (hunk 1 above)
Comment (BupycHuk) [reply .github/workflows/doc.yml:35]: to have less risk on it I would add version flag as well

**Review by BupycHuk — APPROVED** [review]

File: .github/workflows/doc.yml (hunk 1 above)
Comment (ademidoff) [reply .github/workflows/doc.yml]: Done.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by idoqo — APPROVED** [review]

File: docs/api/advisor-api/overview.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
-This section is about API for managing Percona [Advisors](https://docs.percona.com/percona-monitoring-and-management/get-started/advisors.html), and [Advisors Checks](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks/index.html).
+This section describes APIs to deal with Percona [Advisors](https://docs.percona.com/percona-monitoring-and-management/get-started/advisors.html), and [Advisors Checks](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks/index.html).
 
-This section has three main subsections.
-
-- In [List of problems detected by Advisors](failed-checks) you can find API to get information about potential problems with your Infrastructure detected by Advisors.
-- [Percona Advisors and Advisors checks](advisors-and-advisor-checks) lists all Advisors and Advisor Checks available in your PMM.
-- [Changing Advisors and Advisor checks](changing-advisor-checks) will help you automate Advisor checks modification if needed.
+- In [List of problems detected by Advisors](getfailedchecks) you can find API to get information about potential problems with your Infrastructure detected by Advisors.
```
Comment (catalinaadam) [inline docs/api/advisor-api/overview.md:11]:
```suggestion
- The [List of Problems Detected by Advisors](https://github.com/percona/pmm/pull/3207/getfailedchecks) API endpoint offers detailed insights into potential infrastructure issues identified by Advisors.
```

File: docs/api/advisor-api/overview.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+- [List Percona Advisors](ref:listadvisors) lists all Advisors available in your PMM.
```
Comment (catalinaadam) [inline docs/api/advisor-api/overview.md:12]:
```suggestion
- [List Percona Advisors](ref:listadvisors) lists all Advisors available for your PMM instance.
```

File: docs/api/advisor-api/overview.md [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+- [List Advisor Checks](ref:listadvisorchecks) lists all Advisor Checks available in your PMM.
+- [Changing Advisor Checks](ref:changeadvisorchecks) will help you automate Advisor checks, i.e.:
```
Comment (catalinaadam) [inline docs/api/advisor-api/overview.md:14]:
```suggestion
- [Changing Advisor Checks](ref:changeadvisorchecks) will help you automate Advisor Checks, i.e.:
```

File: docs/api/inventory/overview.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+slug: pmm-inventory
+categorySlug: inventory-api
+hidden: 0
+order: 0
+---
+
+This section will provide you with information on how to add, update or remove services, nodes and agents.
```
Comment (catalinaadam) [inline docs/api/inventory/overview.md:9]:
```suggestion
This section guides you on how to add, update, or remove services, nodes, and agents.
```

**Pull request merged**